tests/installation/user_settings.pm

```diff
@@ -38,7 +39,7 @@ sub run {
         send_key $cmd{noautologin};
         assert_screen 'autologindisabled';
     }
-    elsif (check_var('NOAUTOLOGIN', '0')) {
+    elsif (is_sle() && check_var('NOAUTOLOGIN', '0')) {
```

Comment:
hm, I guess it will not be very clear to the reader of this code why we would need the `is_sle`. I guess the problem here is rather https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/products/sle/main.pm#L123 which sets the default for SLE. I wonder how to better encode this. We have the method `default_desktop` which we could apply in a similar fashion for the "autologin/no-autologin" option as well or maybe just another specific variable, e.g. `set_var('NOAUTOLOGIN_DEFAULT', is_sle ? 1 : 0)` in lib/main_common.pm. WDYT? If you do not like it or don't have a better idea let's just keep it as you have it in your PR and merge (-> implicit approval :) ).

Reply:
I already thought about what you mentioned in some way and I found pretty ugly existing code, I would even think about creating a new test module instead of adding more complexity to new one with new magic variables or main.pm adjustment. So, yes, better merge it as it is.